### PR TITLE
Merge Client Side 

### PR DESCRIPTION
**Pull request by choitwao**
Hey guys so I am merging this for now, not very complete but should have no issue setting it up and run it.
Just follow the README!

here are the changes
- Login page with styling and stuffs. Controller is semi-finished and waiting for rest api.
- Dash board (home page): finished some basic styling with header component and body component. There will be a calendar view in the middle (white part) with room list and equipment list (changed when you select different room). I will update a sketch UI design later. 

I am still working on setting up the calendarJS, seems like there is issue with packaging vendor libraries. Plan B I will remove the stupid bower and use only npm. Worst scenerio we can ng-repeat the calendar manually. Also, I disable the test part for now until I figure out why ngMocks is giving me errors with angular 1.6 (the latest version)

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -35,19 +35,21 @@
...
-- `npm run package` to build the web application into `dist` folder
+- `npm run pack` to build the web application into `dist` folder
 
 ### Development
 
-- `npm run package` to package the application into `dist` folder
+- `npm run pack` to package the application into `dist` folder
 - `npm run clean` to delete build folders
 - `npm run start` to run the application on Chrome
 - `npm run test` to run unit tests
```
Comment (adriannadiaz) [inline client/README.md]: npm run test doesn't work


> Caproomster@1.0.0 test /Users/adriannadiaz/PycharmProjects/caproomster/client
> karma start test/test.conf.js

10 03 2017 16:08:54.507:INFO [karma]: Karma v1.5.0 server started at http://0.0.0.0:9876/
10 03 2017 16:08:54.514:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
10 03 2017 16:08:54.523:INFO [launcher]: Starting browser PhantomJS
10 03 2017 16:08:55.537:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket eBA6bbWOEMBuCsN1AAAA with id 26609491
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  TypeError: undefined is not a constructor (evaluating 'angular.module('ngAnimateMock', ['ng'])
    .info({ angularVersion: '1.6.3' })')
  at node_modules/angular-mocks/angular-mocks.js:800
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.078 secs / 0 secs)

npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/Cellar/node/7.6.0/bin/node" "/usr/local/bin/npm" "run" "test"
npm ERR! node v7.6.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! Caproomster@1.0.0 test: `karma start test/test.conf.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Caproomster@1.0.0 test script 'karma start test/test.conf.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Caproomster package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     karma start test/test.conf.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Caproomster
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Caproomster
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/adriannadiaz/PycharmProjects/caproomster/client/npm-debug.log

File: client/README.md [hunk 2]
```diff
@@ -35,19 +35,21 @@
...
 
-- `npm run package` to build the web application into `dist` folder
+- `npm run pack` to build the web application into `dist` folder
 
 ### Development
 
-- `npm run package` to package the application into `dist` folder
+- `npm run pack` to package the application into `dist` folder
 - `npm run clean` to delete build folders
 - `npm run start` to run the application on Chrome
```
Comment (adriannadiaz) [inline client/README.md]: run the application on a web browser, on localhost:8080

Comment (adriannadiaz) [inline client/README.md]: Not just Chrome 😁

**Comment by choitwao** [comment]
@adriannadiaz all reviewed changes are made, can you approve thanks

**Pull request merged**